golang/Makefile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.PHONY: build-go build push
+
+export GCS_BUCKET?=k8s-scale-golang-build
+export GO_COMPILER_PKG?=go1.14.2.linux-amd64.tar.gz
+export GO_COMPILER_URL?=https://dl.google.com/go/$(GO_COMPILER_PKG)
+
```

Comment:
Can we document here (can be as a comment) the mandatory env variable that the script assumes (e.g. K8S_COMMIT)?

Reply:
Done.